static/get_dir.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * require this file to get the timestamp to use in the URL
+ *
+ * $ts = (require 'path/to/static/get_dir.php');
+ * $url = "/min/static/$ts/f=min/quick-test.js";
+ */
+
+return call_user_func(function () {
```

Comment:
would be nice to document behavior of this function, without having to read the code and think what it is doing.